model-optimizer/extensions/front/softplus.py

```diff
@@ -0,0 +1,44 @@
+"""
+ Copyright (C) 2020 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+from extensions.ops.activation_ops import Exp, Log
+from extensions.ops.elementwise import Add
+from mo.front.common.partial_infer.utils import float_array
+from mo.front.common.replacement import FrontReplacementOp
+from mo.front.tf.graph_utils import create_op_node_with_second_input
+from mo.graph.graph import Graph
+
+
+class SoftPlus(FrontReplacementOp):
+    """
+    SoftPlus computes logarithm of (1 + Exp(x)).
+    It replaces SoftPlus operation with Exp -> Add -> Log.
+    """
+    op = 'SoftPlus'
+    enabled = True
+
+    def replace_sub_graph(self, graph: Graph, match: dict):
+        softplus = match['op']
+
+        name = softplus.soft_get('name', softplus.id)
+        exp_node = Exp(graph, {'name': name + '/Exp'}).create_node()
+        add_node = create_op_node_with_second_input(graph, Add, float_array([1.0]), {'name': name + '/Add'})
+        log_node = Log(graph, {'name': softplus.name}).create_node()
```

Comment:
This is not enough to properly set the node name.

You should have here a line like the line 36 in `model-optimizer/extensions/front/AttributedPadToPad.py`